HunterPie.Core/Domain/Process/IProcessManager.cs

```diff
@@ -11,6 +11,10 @@ public interface IProcessManager
         public SystemProcess Process { get; }
         public GameProcess Game { get; }
 
+        /// <summary>Whether the last process exit is a normal exit (rather than crash).</summary>
+        /// <value><c>null</c> if the game has not ever started, or is currently running.</value>
+        public bool? IsExitedNormally { get; }
```

Comment:
I'd name this `HasExitedNormally`, since it's in the past and not present.

Reply:
Actually I was wondering if there could be a better name than mine original one. Your proposal rocks.